app/views/admin/groups/_form.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= simple_form_for([:admin, group]) do |f| %>
+  <%= f.input :name %>
+  <%= f.input :number %>
+  <%= f.association :introduction, collection: Introduction.all, label_method: :to_s %>
```

Comment:
`Introduction.by_start` can be useful, and have 
`scope :by_start, -> { order(start: :desc) }`